CHANGELOG.md

```diff
@@ -30,6 +30,8 @@
 - Added the Go version, OS, and architecture to the output of `operator-sdk version` ([#1863](https://github.com/operator-framework/operator-sdk/pull/1863))
 - Added support for `ppc64le-linux` for the `operator-sdk` binary and the Helm operator base image. ([#1533](https://github.com/operator-framework/operator-sdk/pull/1533))
 - Added new `--version` flag to the `operator-sdk scorecard` command to support a new output format for the scorecard. ([#1916](https://github.com/operator-framework/operator-sdk/pull/1916)
+- Added new `--selector` flag to the `operator-sdk scorecard` command to support filtering scorecard tests based on labels added to each test. ([#1916](https://github.com/operator-framework/operator-sdk/pull/1916)
```

Comment:
IHMO would be easier if we start to avoid create PR's with more than one command in order to try to keep them small and easier to review, easier to track and etc.

Reply:
I tried to keep this PR small by just adding the flags and not the actual implementation of those flags, I added --dry-run last minute since it seemed pretty easy to do without blowing up the PR size.

Reply:
So, IMHO we should NOT move with this PR because it is NOT the impl of a small unit which brings value. As we spoke, I do NOT agree with this approach to merge something incomplete in the master. We can break the tasks, but the small part should bring some value to the product. 

For example, what happens for the users that are using the master branch and try to check/use it since it will appear for them or if we need to a release and it has not finished yet? 

Also, in POV is easier to review and ensure the quality, track, rollback/revert, do cherry-pick if be required and etc if 1 PR means the code implementation, the tests and docs. My suggestion would be make 1 PR for each flag, where should have these artefacts.

However, I will leave it to the team to decide.

Reply:
agreed, I'll remove the dry-run flag from this PR and also add in the full --selector implementation for one review of the whole selector feature, that makes sense.